libraries/api-documenter/ad.cmd

```diff
@@ -0,0 +1,3 @@
+@ECHO OFF
```

Comment:
The name "ad" is kind of too brief.

Reply:
The real executable is "api-documenter".  This ad.cmd file is just used by me to invoke the tool locally when debugging it.

---
In reply to: [145812638](https://github.com/Microsoft/web-build-tools/pull/399#discussion_r145812638) [](ancestors = 145812638)